app/models/shipit/stack.rb

```diff
@@ -528,8 +528,25 @@ def sync_github
       GithubSyncJob.perform_later(stack_id: id)
     end
 
+    alias links_orig links
+    def links
+      persisted_tasks = tasks.select(&:persisted?)
+      return links_orig if persisted_tasks.blank?
+
+      variables = Shipit::TaskCommands.new(persisted_tasks.last).env
+      links_orig.each_with_object({}) do |(name, url), replaced_links|
```

Comment:
`Hash#transform_values` would be much cleaner here.

Reply:
Wow, `Hash#transform_values` is much nicer. Thanks for the suggestion!

Reply:
Addressed by https://github.com/Shopify/shipit-engine/pull/1018/commits/ae342cc1e1b9e5447736efa569d88aa0b67e6ead